lib/client_summarizer.rb

```diff
@@ -0,0 +1,31 @@
+class ClientSummarizer
+  include FiscalYearMixin
+
+  attr_reader :client_namespace, :fiscal_year
```

Comment:
Mostly they are defined per my idea of good public api construction. Since fiscal year, in particular, might be internally derived if not explicitly set, I wanted to make sure it was possible to interrogate the object to discover what was used. I have added tests for that in 588f093